packages/eslint-plugin/tests/docs.test.ts

```diff
@@ -575,6 +576,29 @@ describe('Validating rule metadata', () => {
           rule.meta.docs?.requiresTypeChecking ?? false,
         );
       });
+
+      it('all properties of the rule options schema must include a description', () => {
```

Comment:
😬 I have to be honest, I really don't think a unit test is the right approach here. The issue asks for a lint rule because that's something every ESLint plugin project can use. An internal Jest unit test isn't shareable in that way.

Reply:
Sounds good! I figured

- this could be something to help us move forward without waiting for an upstream update
- as well as potentially better accommodating any potential cases where the options schema is somehow dynamically generated and cannot be statically analyzed by a lint rule

but no worries if you'd prefer not to go this way!